src/__experimental__/components/input/input-presentation-classic.style.js

```diff
@@ -0,0 +1,25 @@
+import { css } from 'styled-components';
```

Comment:
Could probably do with some commentary on why we're extracting out `classic` properties in this way.

Reply:
I think it's best we add this detail to the wiki on the repo, a segment on theming and how to handle classic theming. 

I plan on adding some detail to that over the next few days.